app/views/hosts/select_multiple_puppet_proxy.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= render 'selected_hosts', :hosts => @hosts %>
+
+<%= form_for :puppet, :url => update_multiple_puppet_proxy_hosts_path(:host_ids => params[:host_ids]) do |f| %>
+  <%= selectable_f f,
+    :puppet_proxy_id,
+    [[_("Select desired puppet master"), "disabled"]] +
```

Comment:
It doesn't look like it's possible to clear out the Puppet proxy field, since all I get is the 'Select desired Puppet master' field and Proxy names. It should be able to reset this - so just change the 'Select desired Puppet master' to 'No Puppet proxy'. Please add a test for this behavior too :smile: !